src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/IslReplugSpec.groovy

```diff
@@ -238,7 +238,7 @@ class IslReplugSpec extends HealthCheckSpecification {
 
         and: "Self-loop rule packet counter is incremented and logged in tsdb"
         Wrappers.wait(statsRouterRequestInterval) {
-            switchStats.of(expectedIsl.getSrcSwitch().getDpId(), [FLOW_SYSTEM_PACKETS])
```

Comment:
just to clarify: it seems like  [FLOW_SYSTEM_PACKETS] part is required, is it ok to remove it?

Reply:
No, it's not needed anymore. Getting rid of OpenTSDB makes it possible to retrieve all stats for switch, not the specified list of the ones.